### DISCUSSION TITLE
Is it possible to distribue this wrapper to other people!

### DISCUSSION THREAD

**Discussion by stromhaug-tommy** [Macad3D/Macad3D#41] [General]
Hi,

I have been using your Opencascade wrapper in a lot of cases. Is it possioble for me to give othrer people access to to your  wrapper? A lot of people are having har problems with boolean operations. Open Cascade is very robust in these cases.

**Comment by dipts** [comment]
The OCCT Wrapper is created specifically for Macad3D and not designed for universal use. 

If this is a legal question: This project itself is licensed under the MIT licence, third party components may be subject to other conditions. Whoever benefits is welcome to make use of it, within the terms of the respective licences.

Just like any open source project, I am happy when improvements are fed back into the project.
